build-gradle-plugin/src/main/kotlin/cz/ackee/gradle/util/VariantExt.kt

```diff
@@ -6,8 +6,8 @@ import org.gradle.api.Task
 import org.gradle.configurationcache.extensions.capitalized
 
 /**
- * TODO: Replace this with some more official solution how to get assemble task from variant,
- *  but for now I wasn't able to find it.
+ * TODO: Replace this with `wiredWith` function but for APK and Bundle artifacts it provides
+ *  wrong output path
```

Comment:
This wording is a little confusing I think. What about something like "Replace this with `wiredWith` function once possible AGP bug fixed. Currently `SingleArtifact.APK` provides an incorrect input file path." Or something like that.

Reply:
amended